lib/sqlalchemy/sql/selectable.py

```diff
@@ -4520,7 +4525,7 @@ def _plugin_not_implemented(cls) -> NoReturn:
 
     @classmethod
     def get_column_descriptions(
-        cls, statement: Select[Any]
+        cls, statement: Select[Unpack[Tuple[Any, ...]]]
```

Comment:
**Michael Bayer** (zzzeek) wrote:

tupleany

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986